### DISCUSSION TITLE
Login to tools with selected auth profile/environment

### DISCUSSION THREAD

**Discussion by filcole** [microsoft/powerplatform-vscode#324] [Ideas]
I love the new `pac tool prt|cmd|pd` commands.  Makes updating them so much easier.

It would be fantastic if the current auth profile and environment selected used by pac could be passed through to these tools (perhaps as an option) so that there we did not have to login again after launching each of these tools - this would make a big difference for PRT particularly.

**Comment by davidjenni** [comment]
@filcole it is on my personal wish list as well, but to actually implement that is not trivial since pac CLI would need to share tokens between 2 different apps; the cache locations for pac CLI and PRT/CMT/PD are for historical reasons quite different.

As cool as it would be, for now it does not meet our overall priority bar for other work we have committed to for this quarter.

**Reply by filcole** [reply]
No problems. Fully understand.  Thanks for converting to a discussion.

**Comment by davidjenni** [comment]
In the meantime, I'm converting this to a discussion.

**Comment by petrochuk** [comment]
We added few other commands. I know it doesn't really address your comment but here they are:

Launch Power Platform Admin Center for the current environment
```
pac tool admin
```

Launch Power Apps Maker Portal for the current environment
```
pac tool maker
```
